frontend/talentsearch/resources/js/components/skills/SearchBar.tsx

```diff
@@ -26,10 +26,12 @@ const SearchBar: React.FunctionComponent<SearchBarProps> = ({
     return handleSearch(data.query);
   };
 
+  // no dependencies for usCallback since we need the same instance every time
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   const submitDebounced = useCallback(
     debounce((query: string) => {
       if (query.length >= 2) handleSubmit(onSubmit)();
-    }, 1000),
+    }, 250),
```

Comment:
Good point on this. In fact, the number of skills should never be all that large - can we remove this debounce entirely without performance issues?

Reply:
Yep, it works well without it.  Poor Yoni.  I was the one who asked it to add it in his original PR. 😭

Reply:
Ahaha, it happens. Always refactoring, always improving.